Danyal_Khan/flask_mysql_projs/emailvalidation/server.py

```diff
@@ -0,0 +1,38 @@
+from flask import Flask, request, redirect, render_template, session, flash
+from mysqlconnection import MySQLConnector
+import re
+app = Flask(__name__)
+mysql = MySQLConnector(app,'email')
+EMAIL_REGEX = re.compile(r'^[a-zA-Z0-9.+_-]+@[a-zA-Z0-9._-]+\.[a-zA-Z]+$')
+app.secret_key = 'KeepItSecretKeepItSafe'
+
+@app.route('/')
+def index():
+    query = "SELECT * FROM email"                           
+    email = mysql.query_db(query)
+
+    return render_template('index.html', all_email='email')
```

Comment:
since you don't actually display any email data on your index.html, there's no need to do a query here or pass any data to it